styles/style.css

```diff
@@ -35,10 +31,24 @@ body {
   background-color: #f44d41;
 }
 
-.deleteButton:hover {
+.deleteButtonStyle:hover {
   background-color: #fa1707;
 }
-.editButton {
+.editButtonStyle {
+  margin: 8px;
+  color: white;
+  padding: 4px 8px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+  background-color: #f0d839;
+}
+
+.editButtonStyle:hover {
+  background-color: #ddc10c;
+}
+.doneButtonStyle {
```

Comment:
It's a good convention to avoid px and use rem. 

```suggestion
.editButtonStyle {
  margin: 0.5rem;
  color: white;
  padding: .25rem 0.5rem;
  border: none;
  border-radius: .25rem;
  cursor: pointer;
  transition: background-color 0.3s;
  background-color: #f0d839;
}

.editButtonStyle:hover {
  background-color: #ddc10c;
}
.doneButtonStyle {
```